Content.Server/Entry/EntryPoint.cs

```diff
@@ -110,11 +137,11 @@ public override void Update(ModUpdateLevel level, FrameEventArgs frameEventArgs)
             switch (level)
             {
                 case ModUpdateLevel.PostEngine:
-                {
-                    _euiManager.SendUpdates();
-                    _voteManager.Update();
-                    break;
-                }
+                    {
+                        _euiManager.SendUpdates();
+                        _voteManager.Update();
+                        break;
+                    }
```

Comment:
```suggestion
                {
                    _euiManager.SendUpdates();
                    _voteManager.Update();
                    break;
                }
```
Should be less indent to be consistent with the rest of the codebase.